package.json

```diff
@@ -0,0 +1,81 @@
+{
+  "name": "fec-express-js",
+  "version": "0.1.0",
+  "description": "",
+  "engines": {
+    "node": "8.9.4",
+    "npm": "2.14.2"
```

Comment:
check which version you are actually using and change it here, to avoid future problems with heroku